packages/destination-actions/src/destinations/clevertap/index.ts

```diff
@@ -0,0 +1,48 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import userUpload from './userUpload'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Clevertap( Actions)',
+  slug: 'clevertap',
```

Comment:
Convention is to use `actions-{name}` for the slug

```suggestion
  slug: 'actions-clevertap',
```

Also, `clevertap` is already taken.